### PR TITLE
Remove jupyter widgets

### LINKED ISSUE #62: Move Jupyter Widget example out of widget-tracker
We could move the `jupyter-widgets` example out of the `widget-tracker` folder, since it's more about Jupyter Widgets than about using a `WidgetTracker`.

### PR DESCRIPTION
**Pull request by fcollonval**
Fix #62 

- Remove `jupyter-widgets` example
- Add mention in `kernel-output` about displaying `ipywidgets`
- Improve `kernel-output` to execute arbitrary statement - so it could be an ipywidget (should the preview be one?)

### CODE REVIEW

File: advanced/kernel-output/README.md [hunk 1]
```diff
@@ -214,20 +215,44 @@ async function createPanel(): Promise<ExamplePanel> {
...
-  caption: 'Show Dataframe',
+  label: 'kernel-output: Execute Code',
+  caption: 'Execute Code',
   execute: async () => {
+    // Create the panel if it does not exist
     if (!panel) {
       await createPanel();
     }
-    panel.execute('df');
+    // Ask the user about the statement to be executed
```
Comment (jtpio) [inline advanced/kernel-output/README.md]: Would "prompt" be more adequate here? Something like "Prompt the user for the statement to execute".

File: advanced/kernel-output/README.md [hunk 2]
```diff
@@ -214,20 +215,44 @@ async function createPanel(): Promise<ExamplePanel> {
...
+    const input = await InputDialog.getText({
+      title: 'Code to be executed',
```
Comment (jtpio) [inline advanced/kernel-output/README.md]: Or?

```suggestion
      title: 'Code to execute',
```

**Comment by jtpio** [comment]
Thanks @fcollonval!

**Comment by fcollonval** [comment]
Thanks for the review @jtpio

**Pull request merged**